src/bio/bio_context.c

```diff
@@ -492,8 +505,18 @@ bio_blob_create(uuid_t uuid, struct bio_xs_context *xs_ctxt, uint64_t blob_sz,
 			ba->bca_id, xs_ctxt, DP_UUID(uuid),
 			bma.bma_opts.num_clusters);
 
-		rc = smd_pool_add_tgt(uuid, xs_ctxt->bxc_tgt_id, ba->bca_id,
-				      blob_sz);
+		if (!(flags & BIO_MC_FL_SYSDB) || !bio_is_meta_on_ssd_configured()) {
+			rc = smd_pool_add_tgt(uuid, xs_ctxt->bxc_tgt_id, ba->bca_id,
+					      blob_sz);
+		} else if (st == SMD_DEV_TYPE_META) {
+			ba->bca_inflights = 1;
+			spdk_bs_set_super(bbs->bb_bs, ba->bca_id,
+					  set_super_cb, &bma);
```

Comment:
Should we set INVALID super blob ID when we delete super blob? Or it's done by SPDK already?

Reply:
As discussed, now bio won't check super exist before set, and spdk_bs_set_super() just overwrite previous super blob id, so it is fine to not set invalid when delete super blob.